bindings/stronghold-nodejs/examples/src/tests/storage_test_suite.ts

```diff
@@ -2,17 +2,31 @@ import { stronghold } from '../stronghold';
 import { StorageTestSuite } from '@iota/identity-wasm/node';
 
 // Only verifies that no uncaught exceptions are thrown, including syntax errors etc.
-describe("Test Stronghold Node.js examples", function () {
-    it("Storage Test Suite", async () => {
-        await StorageTestSuite.didCreateTest(await stronghold());
+describe("Test Stronghold Node.js", function () {
+    it("didCreateGenerateKey", async () => {
+        await StorageTestSuite.didCreateGenerateKeyTest(await stronghold());
+    });
+    it("didCreatePrivateKey", async () => {
+        await StorageTestSuite.didCreatePrivateKeyTest(await stronghold());
+    });
+    it("keyGenerate", async () => {
         await StorageTestSuite.keyGenerateTest(await stronghold());
+    });
+    it("keyGenerate", async () => {
         await StorageTestSuite.keyDeleteTest(await stronghold());
```

Comment:
Should be `keyDelete`

```suggestion
    it("keyDelete", async () => {
        await StorageTestSuite.keyDeleteTest(await stronghold());
```